fixed_frame: "odom"
use_mag: true
publish_tf: false
reverse_tf: false
constant_dt: 0.0
publish_debug_topics: false
gain_acc: 0.01
gain_mag: 0.01
do_bias_estimation: true
bias_alpha: 0.01
do_adaptive_gain: true
